delivering/release-process/index.md

```diff
@@ -0,0 +1,63 @@
+---
+layout: default
+title: Release Process
+has_children: true
+parent: How We Deliver
+---
+
+Some of these responsibilities may differ depending on the type of project, please refer to those documents for more specifics.
+
+## Responsibilities
+
+This document outlines high-level responsibilities
+
+### Communication
+
+Communication around releases is important:
+
+- The main channel for communication around releases is the Slack channel for the project in question.
+- Engineers are free to augment Slack with other tools (e.g. Zoom) as they see fit.
+- If you won't be available to verify your changes in RC, communicate this to the team and ask a team member to do this in your absence before merging changes.
+- If a release candidate isn't viable for release, communicate this and what the next step(s) are.
+
+### Individual Engineer
+
+As an engineer you have responsibilities to ensure the work you've done is reliably delivered.
+
+After your code is merged:
+
+- You may wait for the daily automated release candidate.
+- If you have a more urgent need to get your changes released, you may trigger a manual release candidate. You should coordinate with other engineers who have also merged changes to ensure they are available to verify their own work, if applicable.
+
+### Engineering Team
+
+Collectively the team members who have made changes in a release candidate are responsible for coordinating the release.
+
+### Release Manager
+
+When a new release candidate is created, a release manager is designated automatically based on the following logic:
+
+As a general rule to follow our approach of having releases only happen during standard , engineers are selected from a pool of those who work during those hours.
+  
+- For **manually triggered** release candidates, the engineer who triggered it.
+- For **automated daily releases**, an engineer is selected at random from those
+
+
+## When can releases happen?
+
+### Release Candidates
+  
+Engineers are welcome to cut release candidates whenever they need to, even during EST off-hours.
+
+### Production Releases
+
+Production releases should only be done Monday through Friday, 8am-4pm to mitigate regressions or unanticipated production bugs from occurring off-hours.
```

Comment:
`Monday through Thursday (Friday if an urgent or low-impact fix/feature is needed), 8am - 4pm` ?

Reply:
I added some examples below of when we'd make exceptions to this rule, how does that look?

https://github.com/mitodl/handbook/pull/47/files#diff-c23697dc8be32ad70101ce4db3c709e53ca811db619ac6b18503e3ab8c9fd215R56-R60

Reply:
LGTM 👍